vite.config.ts

```diff
@@ -0,0 +1,20 @@
+import react from "@vitejs/plugin-react";
+import { defineConfig } from "vite";
+import EnvironmentPlugin from "vite-plugin-environment";
+import eslint from "vite-plugin-eslint";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react(), tsconfigPaths(), eslint(), EnvironmentPlugin("all")],
+  server: {
+    port: 8080,
+    proxy: {
+      "/api": {
+        target: "https://ll.thespacedevs.com/2.2.0/launch/",
+        changeOrigin: true,
+        rewrite: (path) => path.replace(/^\/api/, ""),
+      },
+    },
+  },
```

Comment:
I tend to like the app opening when the dev environment starts up. Can you add `open: true,` here?